### PR TITLE
es6-operators - Kyle Kettler

### PR DESCRIPTION
**Pull request by kyle-kettler**
Screenshot 2023-06-20 at 5 01 24 PM

### CODE REVIEW

File: es6-operators/main.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+const v4 = [3, 5, 7, 9];
+
+/* TODO:
+ * Demonstrate how to execute an expression only if a variable is truthy.
+ * In one expression, log "v1 is truthy" to the console only if `v1` is truthy.
+ * In one expression, log "v2 is truthy" to the console only if `v2` is truthy.
+ * Expected: v1 message logs, v2 message does not.
+ */
+console.log(v1 && 'v1 is truthy');
+console.log(v2 && 'v2 is truthy');
```
Comment (shawnkost) [inline es6-operators/main.js:14]: Your console log should only fire if the first condition is true. Right now, your 2nd console log is still firing, it's just not printing any value

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Kyle!

**Comment by kyle-kettler** [comment]
Screenshot 2023-06-20 at 5 18 15 PM

Got it fixed!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**